### PR TITLE
completed javascript-ajax

### PR DESCRIPTION
**Pull request by t-doan**
javascript-ajax

### CODE REVIEW

File: javascript-ajax/javascript-ajax-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- What is AJAX?
+
+- What does the AJAX acronym stand for?
+
+- Which object is built into the browser for making HTTP requests in JavaScript?
+
+- What event is fired by `XMLHttpRequest` objects when they are finished loading the data from the server?
+
+- **Bonus Question**: An `XMLHttpRequest` object has an `addEventListener()` method just like DOM elements. How is it possible that they both share this functionality?
+
```
Comment (shawnkost) [inline javascript-ajax/javascript-ajax-quiz-notes.md:16]: Missing the quiz questions

**Review by shawnkost — CHANGES_REQUESTED** [review]
Need to answer the quiz questions!

File: javascript-ajax/javascript-ajax-quiz-notes.md [hunk 2]
```diff
@@ -5,14 +5,19 @@
...
+  Asynchronous JavaScript and XML
 
 - Which object is built into the browser for making HTTP requests in JavaScript?
+  XMLHTTPRequest()
 
 - What event is fired by `XMLHttpRequest` objects when they are finished loading the data from the server?
+  Load
 
 - **Bonus Question**: An `XMLHttpRequest` object has an `addEventListener()` method just like DOM elements. How is it possible that they both share this functionality?
+  Because we are reading data being sent and loaded to our html page
```
Comment (t-doan) [inline javascript-ajax/javascript-ajax-quiz-notes.md:20]: Completed the quiz questions

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**